app.js

```diff
@@ -11,7 +11,7 @@ const spotify = new Spotify();
  * user will be in loop of ECONNRESET [changed address]:80 or recieve false data.
  **/
 try {
-	const path = process.platform === 'win32' ? 'C:\\Windows\\System32\\drivers\\etc\\hosts' : '/hosts/etc';
+	const path = process.platform === 'win32' ? 'C:\\Windows\\System32\\drivers\\etc\\hosts' : '/etc/hosts';
```

Comment:
What about win32, not every installation is at C:\ nor \Windows\. Use %SystemRoot% to your advantage.